Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -152,20 +153,57 @@ namespace AZ
             return m_visScene->GetEntryCount();
         }
 
+        CullingDebugContext& CullingScene::GetDebugContext()
+        {
+            return m_debugCtx;
+        }
+
+        const AzFramework::IVisibilityScene* CullingScene::GetVisibilityScene() const
+        {
+            return m_visScene;
+        }
+
+        static AzFramework::EntityContextId GetEntityContextIdForOcclusion(const AZ::RPI::Scene* scene)
```

Comment:
Since this is a static function in a cpp file add a quick comment explains the purpose of this function.